assets/node_nfs.yaml

```diff
@@ -18,8 +18,13 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccount: manila-csi-driver-node-sa
       priorityClassName: system-node-critical
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
```

Comment:
Same comment about the node selector.

Reply:
Same as above - the nodeSelector has been removed and the additional toleration dropped.